app/helpers/recovery_code_helpers.rb

```diff
@@ -0,0 +1,15 @@
+module RecoveryCodeHelpers
+  def modal_controller_js
```

Comment:
should we call this the `recovery_code_modal` instead of `modal_controller_js` -- since it sounds more generic than it is?